changelog.d/428.change.rst

```diff
@@ -0,0 +1,6 @@
+Added ``@attr.s(collect_by_mro=False)`` argument that if set to ``True`` fixes the collection of attributes from base classes.
```

Comment:
Is it necessary to have two identical change fragments?

Reply:
Yeah that's how towncrier works nowadays I'm afraid. It will detect that it's the same and coalesce them into one entry.

Reply:
Oh wow. I'm curious if it'd work with symlinks...